.github/workflows/deployment_server.yml

```
@@ -0,0 +1,100 @@
+    paths:
+      - "apps/server/**"
+      - ".github/workflows/deployment_server.yml"
+
+jobs:
+  get-version:
+    runs-on: ubuntu-latest
+    outputs:
+      SERVER_VERSION: ${{ steps.get_version.outputs.SERVER_VERSION }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)